docs/changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 ~~~~~~~~~
 
+0.6.0 (March 2023)
+------------------
+
+* Updated pytket version requirement to 1.13.
+* Updated iqm-client version requirement to 11.8.
```

Comment:
From my understanding the update to the client includes some API changes, is that right? Should we point them out here?

Reply:
I was mainly thinking about the new `get_metadata` which looks like it would be available for the public?

Reply:
Yeah that's true, I'll add a line about it.